Solutions/Tomcat/Analytic Rules/TomcatServerErrorsAfterMultipleRequestsFromSameIP.yaml

```diff
@@ -0,0 +1,45 @@
+id: 875da588-4875-11ec-81d3-0242ac130003
+name: Tomcat - Server errors after multiple requests from same IP
+description: |
+  'Detects server errors after multiple requests from same IP address.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ApacheTomcat
+    dataTypes:
+      - TomcatEvent
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
+  - InitialAccess
+relevantTechniques:
+  - T1498
+  - T1190
+  - T1133
+query: |
+  let multi_errors = TomcatEvent
+  | where toint(HttpStatusCode) >= 500 and toint(HttpStatusCode) <= 599

```

Comment:
same here on toint()

Reply:
Query works in Sentinel w/o casting toint().
Casting toint() was added to pass checks. If there is no casting toint() in files then checks fail with error

Reply:
@spsocprime - Are you indicating that this fails checks?  The number needs to be quoted as it is a string -

| where HttpStatusCode >= '500' and HttpStatusCode <= '599'

Reply:
I just created commit adding quotes and removing cast in one of the rules to show you this will not allow to pass checks. You can see screenshots below.
Screenshot#1 Quotes are used
<img width="498" alt="image" src="https://user-images.githubusercontent.com/94110440/159229593-0ac8ffc7-5b89-4b42-8416-f4293086a2ee.png">
Screenshot#2 Error in checks
<img width="814" alt="image" src="https://user-images.githubusercontent.com/94110440/159229526-b84b7968-cbe0-42aa-b94f-eb54efa35e68.png">

If we use casting toint() all check pass w/o errors